src/sandbox/sandbox_linux.go

```diff
@@ -20,6 +21,8 @@ const mdLazytime = 1 << 25
 
 const sandboxDirsVar = "SANDBOX_DIRS"
 
+var sandboxMountDir = core.SandboxDir
```

Comment:
Not sure we need this?

Reply:
Not really, I guess I was trying to make this not reliant on `core.Sandbox` if the variable name would ever change.